runtimes/cpp-17/README.md

```diff
@@ -1,7 +1,7 @@
 
 # C++ Runtime 17
 
-This is the Open Runtime that builds and runs C++ code based on a `alpine-3.15` base image. 
+This is the Open Runtime that builds and runs C++ code based on a `alpine:3.16` base image. 
```

Comment:
File name should also be changed to cpp-17.0

Reply:
Looks like C++ only got major versions: https://en.wikipedia.org/wiki/C%2B%2B17